easybuild/easyblocks/n/neuron.py

```diff
@@ -60,85 +62,133 @@ def extra_options():
         extra_vars = {
             'paranrn': [True, "Enable support for distributed simulations.", CUSTOM],
         }
-        return ConfigureMake.extra_options(extra_vars)
+        extra_vars.update(ConfigureMake.extra_options())
+        extra_vars.update(CMakeMake.extra_options())
+        return extra_vars
 
     def configure_step(self):
         """Custom configuration procedure for NEURON."""
+        if LooseVersion(self.version) < LooseVersion('7.8'):
+            # enable support for distributed simulations if desired
+            if self.cfg['paranrn']:
+                self.cfg.update('configopts', '--with-paranrn')
+
+            # specify path to InterViews if it is available as a dependency
+            interviews_root = get_software_root('InterViews')
+            if interviews_root:
+                self.cfg.update('configopts', "--with-iv=%s" % interviews_root)
+            else:
+                self.cfg.update('configopts', "--without-iv")
+
+            # optionally enable support for Python as alternative interpreter
+            python_root = get_software_root('Python')
+            if python_root:
+                self.with_python = True
+                self.cfg.update('configopts', "--with-nrnpython=%s/bin/python" % python_root)
+
```

Comment:
I didn't see `--with-readline=/readline/install/prefix` option. NEURON has readline+ncurses dependency for interactive interpreter shell. Often configure will find one from the login/build node. So it's safe to provide this option explicitly.

Reply:
This seemed to be automatically detected in my build.

Reply:
ok 👍